src/topp/SpectraMerger.cpp

```diff
@@ -128,10 +128,13 @@ class TOPPSpectraMerger :
 
     PeakMap exp;
     fh.loadExperiment(in, exp, in_type, log_type_);
+    if (exp.getNrSpectra() == 0)
+    {
+      throw Exception::InvalidSize(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION, exp.getNrSpectra());
+    } 
```

Comment:
is this helpful in automated pipelines?
It will force the user to remove the offending file manually.

Reply:
I have to admit, I never had any empty files in normal processing pipelines.
I would assume that the spectra merger should fail in this case and maybe even go so far
to require that every loaded spectra file is not empty.

Reply:
ok, lets keep as is.